### ISSUE TITLE
error messages on `set_dhs_credentials()` could be more helpful

### ISSUE THREAD

**Issue by sumalibajaj** [OJWatson/rdhs#25]
```
> set_dhs_credentials("/Users/sumalibajaj/Documents/Imperial/Rcode/Data/RDHS/credentials.txt",
+                     "/Users/sumalibajaj/Documents/Imperial/Rcode/Data/RDHS/")
Error in FUN(X[[i]], ...) : subscript out of bounds
```

**Comment by sumalibajaj** [comment]
```
> set_dhs_credentials("/Users/sumalibajaj/Documents/Imperial/Rcode/Data/RDHS/credentials.txt",
+                     "/Users/sumalibajaj/Documents/Imperial/Rcode/Data/RDHS/")
Error in FUN(X[[i]], ...) : subscript out of bounds
In addition: Warning messages:
1: In strsplit(readLines(filename, warn = FALSE), "=") :
  input string 1 is invalid in this locale
2: In strsplit(readLines(filename, warn = FALSE), "=") :
  input string 2 is invalid in this locale
3: In strsplit(readLines(filename, warn = FALSE), "=") :
  input string 3 is invalid in this locale
```

**Comment by jeffeaton** [comment]
The vignette may be interpreted as slightly misleading:

> * email=“dummy@gmail.com”
> * password=“dummypass”
> * project=“Dummy Project”

 It makes it appear you need quotation marks, and in some encodings it appears these might be problematic. Suggest revised documentation to something like

```
email=<dummy@gmail.com>
password=<dummypass>
project=<Dummy Project>
```

And also improved error handling when convenient.

**Comment by OJWatson** [comment]
okay cool will change that in the docs and also add the recommendation of using notepad etc rather than word or similar

**Comment by OJWatson** [comment]
this will hopefully be fixed in the v0.5 when we move to the configs, so closing for now
